templates/taxi/manufacturer_list.html

```diff
@@ -1,50 +1,29 @@
 {% extends "base.html" %}
+{% load static %}
+
+{% block title %}Manufacturer List{% endblock %}
 
 {% block content %}
-  <h1>
+<h1>
+    <link rel="stylesheet" href="{% static 'css/style.css' %}">
     Manufacturer List
-    <a href="{% url 'taxi:manufacturer-create' %}" class="btn btn-primary link-to-page">
-      Create
+    <a href="{% url "taxi:manufacturer-create" %}" class="btn btn-primary link-to-page">
+        Create
     </a>
-  </h1>
+</h1>
 
-  {% if manufacturer_list %}
-    <table class="table">
-      <tr>
-        <th>ID</th>
-        <th>Name</th>
-        <th>Country</th>
-        <th>Update</th>
-        <th>Delete</th>
-      </tr>
+{% include "includes/search_form.html" with search_placeholder="manufacturer name" %}
 
-      {% for manufacturer in manufacturer_list %}
-        <tr>
-          <td>
-              {{ manufacturer.id }}
-          </td>
-          <td>
-              {{ manufacturer.name }}
-          </td>
-          <td>
-              {{ manufacturer.country }}
-          </td>
-          <td>
-              <a href="{% url 'taxi:manufacturer-update' pk=manufacturer.id %}">
-                Update
-              </a>
-            </td>
-            <td>
-              <a style="color: red"
-                href="{% url 'taxi:manufacturer-delete' pk=manufacturer.id %}">
-                Delete
-              </a>
-            </td>
-        </tr>
-      {% endfor %}
-    </table>
+{% if manufacturer_list %}
+    <ul>
+        {% for manufacturer in manufacturer_list %}
+            <li>{{ manufacturer.name }}</li>
+        {% endfor %}
+    </ul>
+{% else %}
+    <p>No manufacturers found.</p>
+{% endif %}
```

Comment:
This block checks if 'manufacturer_list' is not empty and displays a list of manufacturers. However, since you are including 'table_list.html' which also uses 'manufacturer_list', this block might be redundant. Consider whether both the unordered list and the table are needed, or if one should be removed.